Gemfile.lock

```diff
@@ -0,0 +1,97 @@
+PATH
+  remote: .
+  specs:
+    just-the-docs (0.5.4)
+      jekyll (>= 3.8.5)
+      jekyll-seo-tag (>= 2.0)
+      rake (>= 12.3.1)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.8.4)
+      public_suffix (>= 2.0.2, < 6.0)
+    colorator (1.1.0)
+    concurrent-ruby (1.2.2)
+    em-websocket (0.5.3)
+      eventmachine (>= 0.12.9)
+      http_parser.rb (~> 0)
+    eventmachine (1.2.7)
+    faraday (2.7.10)
+      faraday-net_http (>= 2.0, < 3.1)
+      ruby2_keywords (>= 0.0.4)
+    faraday-net_http (3.0.2)
+    ffi (1.15.5)
+    forwardable-extended (2.6.0)
+    google-protobuf (3.23.4-x86_64-linux)
+    http_parser.rb (0.8.0)
+    i18n (1.14.1)
+      concurrent-ruby (~> 1.0)
+    jekyll (4.3.2)
+      addressable (~> 2.4)
+      colorator (~> 1.0)
+      em-websocket (~> 0.5)
+      i18n (~> 1.0)
+      jekyll-sass-converter (>= 2.0, < 4.0)
+      jekyll-watch (~> 2.0)
+      kramdown (~> 2.3, >= 2.3.1)
+      kramdown-parser-gfm (~> 1.0)
+      liquid (~> 4.0)
+      mercenary (>= 0.3.6, < 0.5)
+      pathutil (~> 0.9)
+      rouge (>= 3.0, < 5.0)
+      safe_yaml (~> 1.0)
+      terminal-table (>= 1.8, < 4.0)
+      webrick (~> 1.7)
+    jekyll-github-metadata (2.16.0)
+      jekyll (>= 3.4, < 5.0)
+      octokit (>= 4, < 7, != 4.4.0)
+    jekyll-sass-converter (3.0.0)
+      sass-embedded (~> 1.54)
+    jekyll-seo-tag (2.8.0)
+      jekyll (>= 3.8, < 5.0)
+    jekyll-watch (2.2.1)
+      listen (~> 3.0)
+    kramdown (2.4.0)
+      rexml
+    kramdown-parser-gfm (1.1.0)
+      kramdown (~> 2.0)
+    liquid (4.0.4)
+    listen (3.8.0)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    mercenary (0.4.0)
+    octokit (6.1.1)
+      faraday (>= 1, < 3)
+      sawyer (~> 0.9)
+    pathutil (0.16.2)
+      forwardable-extended (~> 2.6)
+    public_suffix (5.0.3)
+    rake (13.0.6)
+    rb-fsevent (0.11.2)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    rexml (3.2.6)
+    rouge (4.1.2)
+    ruby2_keywords (0.0.5)
+    safe_yaml (1.0.5)
+    sass-embedded (1.64.1-x86_64-linux-gnu)
+      google-protobuf (~> 3.23)
+    sawyer (0.9.2)
+      addressable (>= 2.3.5)
+      faraday (>= 0.17.3, < 3)
+    terminal-table (3.0.2)
+      unicode-display_width (>= 1.1.1, < 3)
+    unicode-display_width (2.4.2)
+    webrick (1.8.1)
+
+PLATFORMS
+  x86_64-linux
```

Comment:
```suggestion
PLATFORMS
  arm64-darwin-21
  x86_64-linux
```

Reply:
I'm sorry, I'm not taking this in right now. Highest priority is on the pipeline, and it currently seems to build fine, even on darwin. Adding your suggestion to my local version here basically breaks bundler (for whatever reason). 

We should take it as it is now, and extend it later when my local issues with your change are sorted out.

Reply:
Hm, while I understand the feeling of urgency, I don't think we should skip over items that *are* important and that we should get right - e.g. what version of bundler we're testing with, that this works for our primary theme developers, etc. I also don't think it's accurate to say that our pipeline is *broken* (this would be different if we were encountering a bug in CI that was preventing us from doing further development).

Reply:
> Adding your suggestion to my local version here basically breaks bundler (for whatever reason).

I had the opposite problem: when I switched just-the-docs-tests to use GH Actions, the attempt to build on GitHub was persistently failing. It turned out that I needed to add the platform `x86_64-linux` to my `Gemfile.lock`.

I'd been following the usual routine to create the `Gemfile.lock`: `rm Gemfile.lock; bundle install`. Running on macOS (with Apple M2 chips), the only platform that `bundle install` includes is `arm64-darwin-22`. The site built okay locally, but on GH Pages the build always failed.

The solution was to create `Gemfile.lock` differently:
```sh
rm Gemfile.lock; bundle lock --add-platform x86_64-linux
```

With hindsight, the error report from the failing runs on GH was clear enough, but it wasn't obvious to me (as a new user of Actions) whether the error was in my files or in the action itself.

Reply:
Your solution is correct.

Insights: https://www.moncefbelyamani.com/understanding-the-gemfile-lock-file/#platforms